problems/sevenBoom.js

```diff
@@ -4,7 +4,12 @@
 * @param {number} n - The number to count up to
 * @returns {number[]} - An array matching the pattern described above
 */
-function sevenBoom() {
+function sevenBoom(num) {
+
+    let output = num.every((el) => {
```

Comment:
Num is a number. every is an array method. We can't call every on a number. Instead we should use a plain old for loop from 1 to num. And push things to an array depending on the condition.